.github/workflows/ci.yml

```diff
@@ -99,7 +99,7 @@ jobs:
         export KUBECONFIG="${HOME}/.kube/config"
         make ct TEST_CHARTS=charts/kubedb
         echo
-        kubectl delete crds -l app=kubedb
+        kubectl delete crds -l app.kubernetes.io/name=kubedb,app.kubernetes.io/instance=kubedb
```

Comment:
We have not updated the annotations on crds. Should we?

https://github.com/kubedb/apimachinery/blob/d99a100155c1b3d4d7d49882309b03345aaf5493/Makefile#L199